### PR TITLE
Axe context

### PR DESCRIPTION
**Pull request by ahuth**
Add a parameter for Axe context. #53 

Useful for excluding parts of the page.

### CODE REVIEW

File: src/ProcessedStory.ts [hunk 1]
```diff
@@ -75,6 +80,14 @@ export default class ProcessedStory {
     return this.parameters.runOptions;
   }
 
+  /**
+   * Context passed to `axe.run`.
+   * @see https://www.deque.com/axe/core-documentation/api-documentation/#context-parameter
+   */
+  get context() {
+    return this.parameters.context;
```
Comment (booc0mtaco) [inline src/ProcessedStory.ts:88]: Not sure if this is checked elsewhere, but is `.parameters` guaranteed to exist here? (assuming typescript would be complaining if not)

**Review by booc0mtaco — APPROVED** [review]

File: src/ProcessedStory.ts (hunk 1 above)
Comment (ahuth) [reply src/ProcessedStory.ts:88]: Yeah, it's initialized in this object's constructor

File: src/ProcessedStory.ts [hunk 2]
```diff
@@ -42,6 +44,9 @@ export default class ProcessedStory {
         rawStory.parameters?.axe?.runOptions,
         rawStory,
       ),
+      context: normalizeContext(rawStory.parameters?.axe?.context, rawStory) as
+        | Context
+        | undefined,
```
Comment (ahuth) [inline src/ProcessedStory.ts:49]: Note the `as`. I didn't feel like creating a Zod schema for all the permutations the context object can take, so left it just checking the general shape.

**Comment by ahuth** [comment]
Thanks 🙏

**Pull request merged**